market/orderTypes/market.ts

```
@@ -0,0 +1,55 @@
+import { getAllCollectibles, getBuyActiveOrdersByCollectibleAndType, getSellActiveOrdersByCollectibleAndType } from "../../database";
+import { Order } from "../../types/Order";
+import { OrderLogic } from "../../types/OrderLogic";
+
+
+
+const market: OrderLogic = {
+    id: 'MARKET',
+    updateAll: function() {
+        const collectibles = getAllCollectibles()
+        
+        for(let i in collectibles)
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'i' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)